docs/using-easybuild.md

```diff
@@ -752,3 +757,19 @@ Dry run: printing build status of easyconfigs and dependencies
 
     Modifying the software version does **not** trickle down the entire software stack, even when combined with `--robot`,
     since a software version is tied to a particular software package.
+
+## Overview of basic LMOD commands  {: #lmod_overview }
```

Comment:
I don't think these commands are specific to Lmod, I'm pretty sure they are all supported by other module tools

Reply:
that's right, changed.